code/API_definitions/kyc-match.yaml

```
@@ -51,19 +52,16 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-
-  version: wip
-
-
+  version: 0.2.0-rc.1
+externalDocs:
+  description: Product documentation at Camara.
+  url: https://github.com/camaraproject/
 servers:
-  - url: '{apiRoot}/{basePath}'
+  - url: '{apiRoot}/kyc-match/v0.2'

```

That would be (for the release candidate):
```suggestion
  - url: '{apiRoot}/kyc-match/v0.2rc1'

```